src/schemas/json/workflows.json

```
@@ -259,6 +259,48 @@
         },
         "for": {
           "$ref": "#/definitions/for"
+        },
+        "parallel": {
+          "type": "object",
+          "description": "Run branches or iterations in parallel",
+          "additionalProperties": false,
+          "properties": {
+            "exception_policy": {
+              "description": "determines action of other branches when an exception occurs",
+              "type": "string",
+              "enum": ["continueAll"]
+            },
+            "shared": {
+              "description": "list of shared variables",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "concurrency_limit": {
+              "type": "integer",
+              "description": "upper limit for branches/iterations to perform concurrently"
```

```suggestion
              "description": "An upper limit for branches/iterations to perform concurrently",
              "type": "integer"
```

Optional fix: key order consistency violated: move `type` one key below.
Optional fix: `An upper` may sound better